app/src/main/res/layout/item_rolled_dice.xml

```diff
@@ -6,53 +6,53 @@
     android:layout_height="wrap_content"
     android:paddingVertical="4dp">
 
-
     <ImageView
-        android:id="@+id/rolledDiceIcon"
-        android:layout_width="64dp"
-        android:layout_height="64dp"
+        android:id="@+id/rolled_dice_icon"
+        android:layout_width="@dimen/rolled_die_image_size"
+        android:layout_height="@dimen/rolled_die_image_size"
+        android:contentDescription="@string/icon_dice_type"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:srcCompat="@drawable/nav_menu_d20_icon"
-        app:tint="@color/white"
-        android:contentDescription="@string/icon_dice_type" />
+        app:tint="@color/white" />
 
     <TextView
-        android:id="@+id/rolledDiceText"
-        android:textStyle="bold"
-        android:textSize="16sp"
+        android:id="@+id/rolled_dice_text"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="16dp"
+        android:layout_marginStart="@dimen/view_vertical_margin_medium"
```

Comment:
Always extract your DP and SP measurements. Give them a good name so people know what they are doing for your layout.